src/js/stores/LocaleStore.js

```diff
@@ -30,6 +31,8 @@ var headerLocaleMap = {
   'pt-BR': 'pt_BR'
 };
 
+var supportedLocalesList = _.values(langLocaleMap).concat(_.values(headerLocaleMap));
```

Comment:
there are some duplicates here -- for instance `zh_CN` -- if we just add the values of each. can you convert to a set? or perhaps just make them all keys in an object?

also we dont need the underscore here since i polyfill

```
var allLocales = {};
Object.values(langLocaleMap).concat(Object.values(headerLocaleMap)).map(locale => allLocales[locale] = true);

// later
return Object.keys(allLocales)
```